modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -302,6 +207,106 @@ def buildProject(Map options = [:]) {
   }
 }
 
+def nonDockerBuildTasks(options, jobName, repoName) {
```

Comment:
I don't think the `nonDocker` prefix is adding much here, it might just cause confusion.

Reply:
Yeah I think it's a bad name, I think buildTasks has it conflicting too much with buildProject and it needs something to differentiate it from the docker equivalent. But something better escapes me.